brainmap-cms/templates/snippets/site-css.html

```diff
@@ -0,0 +1,4 @@
+{% load staticfiles %}
+
+{# TODO: FP-1652: Move to template that extends core_cms one #}
+<link rel="stylesheet" href="{% static 'brainmap-cms/css/build/site.css' %}">
```

Comment:
This is a new solution*; https://ecepalliance.org was the first site to use it ([source](https://github.com/TACC/Core-CMS-Resources/blob/9aa0304/ecep-cms/templates/snippets/site-css.html)).

<sup>* Easier than creating template. Proper override and extension not possible in TACC/Core-CMS. To go down the rabbit hole, see [FP-1652](https://jira.tacc.utexas.edu/browse/FP-1652).</sup>